sass/base/components/partners/_partners.scss

```diff
@@ -1,6 +1,11 @@
-// Partners
+// Partners Grid
```

Comment:
Hmmm, i don't think this is right:

We have Partner **pages** : https://www.sportrelief.com/partners/sainsburys
**and** Partner _grids_: https://www.sportrelief.com/partners

Maybe it's best to make Partners Grid its own component and move the styles in this file to there?